### DISCUSSION TITLE
Signaler et filter les opérateurs alternatifs des opérateurs producteur

### DISCUSSION THREAD

**Discussion by JasonMarechal25** [JC144/EDF_Simulateur_Prix#93] [Ideas]
Bonjour
A titre personnel mais c'est surement le cas d'autres personne, je préfère payer un peu plus cher edf ou d'autres opérateurs qui produisent réellement de l'électricité, plutôt que des opérateurs qui ne font qu'acheter/revendre au dépends d'EDF par exemple.

Est-ce qu'il serait ok d'avoir un tel filtre sur le site ?
Le cas échéant j'imagine que la plus grande difficulté sera de définit correctement chaque opérateur.

**Comment by JC144** [comment]
J'ai commencé une ébauche pour avoir une case à cocher "Prix communautaire" sous lequel je compte mettre tous les tarifs non EDF.

Cela me permettrait de séparer tous ces tarifs que je n'ai pas forcément le temps de maintenir et qui ne jouent pas le jeu de faciliter le partage de leurs données même après les avoir contactés. 

Mais j'ai manqué de temps ces dernières semaines 😅

**Comment by zdimension** [comment]
Je viens de voir l'ajout de la case en question et je pense qu'il pourrait être intéressant et l'inverser ; j'ai récemment conseillé le comparateur à quelques collègues et certains ont m'ont répondu que c'était un site sympa mais qu'il ne listait que peu de forfaits... logique puisque si on ne touche à rien ça n'affiche que les tarifs EDF. La case est une bonne idée mais je pense que par défaut (i.e. si on ne touche à aucune case ni aucun champ) le comparateur devrait lister tous les forfaits, et seulement si on le désire, que ça n'affiche que les tarifs EDF.

**Reply by JC144** [reply]
Ma problématique actuelle c'est que les opérateurs ne jouent pas le jeu malgré les échanges que j'ai eu avec eux. Quelques uns devaient me fournir un accès à leur API ou m'inclure dans leur mailing list, mais cela ne s'est pas fait.
Je ferai sûrement une passe de relance à la rentrée.

Actuellement, la maintenance des tarifs est chronophage, je ne peux pas garantir qu'ils soient tous à jour au-delà de ceux d'EDF. D'où la séparation communautaire pour l'indiquer.

**Reply by zdimension** [reply]
Je comprends la problématique, mais là c'est plutôt une question d'UI. Une case, les gens passent à côté. Peut-être faudrait-il un choix obligatoire ? Voici un mockup (à droite le bouton est grisé tant qu'on a pas coché l'une ou l'autre des cases) :
![image](https://github.com/user-attachments/assets/61369b2f-2bec-439e-8c76-cca43a0301c8)

Comme ça les utilisateurs doivent se poser la question, et ça évite qu'ils lisent trop vite et pensent que le comparateur n'a que les tarifs EDF.

**Reply by JC144** [reply]
Pas mal en effet !
Je regarderai ça !

**Reply by JC144** [reply]
J'ai mis à jour, vous me direz ce que vous en pensez.
Je trouve le radio button de bootstrap bien trop clair :/

**Reply by ccoVeille** [reply]
Il manque un id sur les radios, les labels ne sont pas cliquables malgré le for= que tu as mis

**Reply by JC144** [reply]
Fixé ! merci
